src/backing/client/client_factory.cc

```diff
@@ -41,12 +41,11 @@ constexpr char kDefaultApiEndpoint[] = "cloudkms.googleapis.com";
 
 std::unique_ptr<Client> MakeDefaultClient(
     absl::optional<std::chrono::milliseconds> timeout) {
-  auto credentials = grpc::GoogleDefaultCredentials();
-  auto stub = grpc_client::CreateKeyManagementServiceStub(kDefaultApiEndpoint,
-                                                          credentials);
-
-  auto clock = std::make_shared<client::SystemClock>();
-  auto factory = grpc_client::CreateClientContextFactory(timeout, clock);
+  auto stub = grpc_client::CreateKeyManagementServiceStub(
```

Comment:
I think if you just move these into `absl::make_unique<GrpcClient>(…)` then you don't even need `std::move`